src/main/java/org/codeforamerica/messaging/models/Template.java

```diff
@@ -0,0 +1,34 @@
+package org.codeforamerica.messaging.models;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import jakarta.persistence.*;
+import jakarta.validation.constraints.NotBlank;
+import lombok.*;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import java.time.OffsetDateTime;
+import java.util.List;
+
+@Entity
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@ToString(onlyExplicitlyIncluded = true)
+public class Template {
+    @Id
+    @GeneratedValue(strategy= GenerationType.IDENTITY)
+    private Long id;
+    @NotBlank
+    @Column(unique=true)
+    @ToString.Include
+    String name;
+    @JsonIgnore
```

Comment:
Not a biggie, but we would want the variants to be serialized to JSON as well, especially when we implement the `GET /templates/{id}` endpoint.